serve.go

```diff
@@ -117,7 +117,11 @@ func (vs *version) serveNotFound(w http.ResponseWriter) {
 
 func (vs *version) serveError(w http.ResponseWriter, key string, err error) {
 	log.Printf("Error fetching value for /%s/%s: %s\n", vs.db.name, key, err)
-	w.WriteHeader(http.StatusInternalServerError)
+	if err == errRequestCanceled {
```

Comment:
We already check some errors in `serveProxied` so I think it'd make more sense to add this there?

Reply:
https://github.com/stripe/sequins/blob/master/serve.go#L77